standalone/test/integration_tests/integrationtest_udp_client.cpp

```diff
@@ -62,18 +62,43 @@ class UdpClientTests : public testing::Test
 protected:
   MockUDPServer mock_udp_server_{ UDP_MOCK_PORT, std::bind(&UdpClientTests::receivedUdpMsg, this, _1, _2) };
 
-  std::unique_ptr<UdpClientImpl> udp_client_{ new UdpClientImpl(std::bind(&UdpClientTests::handleNewData, this, _1, _2),
-                                                                std::bind(&UdpClientTests::handleError, this, _1),
-                                                                HOST_UDP_PORT,
-                                                                inet_network(UDP_MOCK_IP_ADDRESS.c_str()),
-                                                                UDP_MOCK_PORT) };
+#if BOOST_VERSION > 107000
```

Comment:
Do we need to test with different boost versions?

Reply:
This does something different. Depending on the BOOST_VERSION the API is slightly different. Since the tests run both on Ubuntu 18 (Boost 1.65) / Ubuntu 20 (Boost 1.71) /  Win 10 (Boost 1.75 for now) this defines are needed.